Runtime/Editor/VisualElements/LootLocker MainWindow/LootLockerMainWindow.cs

```diff
@@ -0,0 +1,503 @@
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+using LootLocker.Extension.Requests;
+using LootLocker;
+using System.Linq;
+#if UNITY_EDITOR && UNITY_2021_3_OR_NEWER
+using LootLocker.Admin;
+
+public class LootLockerMainWindow : EditorWindow
+{
+
+    Color LLGreen = new Color(0.10980392156862745f, 0.9098039215686274f, 0.42745098039215684f);
+    Color LLForestGreen = new Color(5, 39, 20);
+
+    enum Page
+    {
+        OrganisationPage,
+        GamePage,
+        GameOptionPage,
+        ApiPage,
+    }
+
+    public enum ContentType
+    {
+        Organisations,
+        Games,
+        GameOptions,
+    }
+
+    Page currentPage;
+
+    private User activeUser;
+    private Organisation activeOrganisation;
+    private Game activeGame;
+
+
+    [SerializeField]
+    private VisualTreeAsset m_VisualTreeAsset = default;
+
+    private VisualElement tabWindow, root;
+    
+    private StyleSheet sheet;
+
+    private Label listHeader, gameTitle, userName, activeWindowLabel;
+
+    private DropdownField keyEnvironment, gameEnvironment;
+
+    private TextField keyName;
+
+    private ScrollView keyList, optionList;
+
+    private Button createKeyButton, returnBtn;
+    
+    [MenuItem("LootLocker/Logout", priority = 10)]
+    public static void Logout()
+    {
+        EditorPrefs.DeleteAll();
+        StoredUser.current.RemoveUser();
+    }
+
+    [MenuItem("LootLocker/Settings", priority = 1)]
+    public static void OpenMenu()
+    {
+
+        LootLockerMainWindow wnd = GetWindow<LootLockerMainWindow>();
+        wnd.titleContent = new GUIContent("LootLocker");
+        if (StoredUser.current.user != null)
+        {
+            LootLockerConfig.current.adminToken = EditorPrefs.GetString("LootLocker.AdminToken");
+            wnd.LoadLootLockerMainMenu(StoredUser.current.user);
+        }
+        else
+        {
+            LootLockerWizard.LoadLogin();
+        }
+
+    }
+    public void LoadLootLockerMainMenu(User user)
+    {
+        LootLockerServerManager.CheckInit();
```

Comment:
I wont as this will only run if you either press the window option or login through the login window, changing it up might cause more issues than solve :P